mypy/test/testfinegrained.py

```diff
@@ -81,22 +81,13 @@ def run_case(self, testcase: DataDrivenTestCase) -> None:
             f.write(main_src)
 
         options = self.get_options(main_src, testcase, build_cache=False)
-        for name, _ in testcase.files:
-            if 'mypy.ini' in name:
-                config = name  # type: Optional[str]
-                break
-        else:
-            config = None
-        if config:
-            parse_config_file(options, config)
+        build_options = self.get_options(main_src, testcase, build_cache=True)
         server = Server(options)
 
+        build_steps = self.get_build_steps(main_src)
```

Comment:
The naming is not very clear. Maybe rename to `num_regular_incremental_steps` or something.